### ISSUE TITLE
Accessibility: MAS40B: Inspect: Name Property of the Files opened Starts with "tab" and Localised Control Type is missing for these files

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/vscode#17176]
VS Code Version:1.6.1
OS Version:1607

**Steps to Reproduce:**
1.Open Visual studio code editor.
2. Open any file from Explorer Side Bar.
3. Navigate to the Name of the opened file and Observe Name and Localised Control Type Property in Inspect.

**Expected Result:**
'Name Property of the Files should contain only the file name displayed and Localised Control Type should be "Tab"
1. Name Property should be "tasks.json"
for file tasks.json which is opened and Localised Control Type should be "Tab"
Narrator Behavior:
Narrator should read as "tasks.json, tab "
NVDA Behavior:
1. NVDA should read as "tasks.json, tab " on Keyboard Navigation and 
on mouse hover.

**Actual Result:**
'Name Property of the Files opened Starts with "tab" and Localised Control Type is missing for these files
1. Name Property is "tab,tasks.json"
for file tasks.json which is opened
Narrator Behavior:
Narrator reads as "tab,tasks.json"
NVDA Behavior:
1. NVDA doesn't read anything on Keyboard Navigation 
2. NVDA reads the Text displayed "tasks.json" on mouse hover.

**User Impact:**
Users relying on screen readers will gets confused if it's announced as "tab,tasks.json" or if it's not announced at all.

![name property of the files opened starts with tab and localised control type is missing for these files](https://cloud.githubusercontent.com/assets/22956130/21184912/8c87b6d8-c234-11e6-9da6-2e93dd07fc1b.jpg)

**Comment by WiproPSAccTest** [comment]
@bpasero The issue is fixed in vs code editor Insider Build.
